events/ready.js

```diff
@@ -1,3 +1,14 @@
+const { ActivityType } = require("discord.js");
+
 module.exports = (client) => {
     console.log(`Ready as ${client.user.tag} to serve in ${client.channels.cache.size} channels on ${client.guilds.cache.size} servers, for a total of ${client.users.cache.size} users.`);
-};
\ No newline at end of file
+    
+    const activities = [
+        `Giveaways in ${client.guilds.cache.size} guilds`,
+    ];
```

Comment:
```suggestion
        `Giveaways in ${client.guilds.cache.size} guilds`,
        `${client.giveawaysManager.giveaways.filter((g) => !g.ended).length} active giveaways`
    ];
```
so it actually changes over time

Reply:
That's kinda cool too I will add it 👍 ❤️